pepper-apis/docs/specification/src/endpoints/admin.studies.participant-lookup-by-guid.yml

```diff
@@ -0,0 +1,30 @@
+get:
+  operationId: participantLookupByGuid
+  tags:
+    - Study Admin
+  summary: ParticipantLookup
+  description: |
+    Lookup a participant by participant guid (user guid).
+
+    It should return a single record with a participant info.
+    If nothing is found then error 404 is returned.
+
+    Note: requires study admin ID token.
+  parameters:
+    - $ref: '../pepper.yml#/components/parameters/userGuid'
+    - $ref: '../pepper.yml#/components/parameters/studyGuid'
```

Comment:
I think it's better to sort this in the same order as how the parameters show up in the path, so it should be `studyGuid` then `userGuid`.

Reply:
Fixed